app/src/main/res/layout/view_unlock.xml

```diff
@@ -45,10 +44,85 @@
         android:gravity="center_horizontal"
         android:lines="2"
         android:maxLines="2"
-        android:text="@string/actions_enter_passcode_or_fingerprint" />
+        android:text="@string/actions_enter_passcode_or_fingerprint"
+        android:visibility="gone" />
 
     <com.saltedge.authenticator.widget.passcode.PasscodeInputView
         android:id="@+id/passcodeInputView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/dp_115" />
+
+    <LinearLayout
+        android:id="@+id/resetPasscodeLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/backImageView"
```

Comment:
backActionView

Reply:
add view type (e.g. ImageView) to View ID name, if it content is changing in application and knowledge of type will help to developer.